content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -97,38 +96,40 @@ By default all binaries are downloaded when you install Prisma CLI. There are ho
 
 ### <inlinecode>PRISMA_BINARIES_MIRROR</inlinecode>
 
-`PRISMA_BINARIES_MIRROR` can be used to specify a custom CDN endpoint (or any server) to download the engines binaries for the CLI/Client. The default value is `https://binaries.prisma.sh`.
+`PRISMA_BINARIES_MIRROR` can be used to specify a custom CDN (or server) endpoint to download the engines binaries for the CLI/Client. The default value is `https://binaries.prisma.sh`.
+
+See [`Navigating a corporate firewall`](../../concepts/overview/under-the-hood#navigating-a-corperate-firewall-with-prisma_binaries_mirror) for a conceptual overview of how to use this environment variable.
 
 ### <inlinecode>PRISMA_MIGRATION_ENGINE_BINARY</inlinecode>
 
 `PRISMA_MIGRATION_ENGINE_BINARY` is used to set a custom location for your own migration engine binary.
 
 ```js
-PRISMA_MIGRATION_ENGINE_BINARY=custom/my-migration-engine-unix
+PRISMA_MIGRATION_ENGINE_BINARY = custom / my - migration - engine - unix
 ```
 
 ### <inlinecode>PRISMA_QUERY_ENGINE_BINARY</inlinecode>
 
 `PRISMA_QUERY_ENGINE_BINARY` is used to set a custom location for your own query engine binary.
 
 ```js
-PRISMA_QUERY_ENGINE_BINARY=custom/my-query-engine-unix
+PRISMA_QUERY_ENGINE_BINARY = custom / my - query - engine - unix
 ```
 
 ### <inlinecode>PRISMA_INTROSPECTION_ENGINE_BINARY</inlinecode>
 
 `PRISMA_INTROSPECTION_ENGINE_BINARY` is used to set a custom location for your own introspection engine binary.
 
 ```js
-PRISMA_INTROSPECTION_ENGINE_BINARY=custom/my-introspection-engine-unix
+PRISMA_INTROSPECTION_ENGINE_BINARY = custom / my - introspection - engine - unix
 ```
 
 ### <inlinecode>PRISMA_FMT_BINARY</inlinecode>
 
 `PRISMA_FMT_BINARY` is used to set a custom location for your own format engine binary.
 
 ```js
-PRISMA_FMT_BINARY=custom/my-custom-format-engine-unix
+PRISMA_FMT_BINARY = custom / my - custom - format - engine - unix
```

Comment:
these edits are broken

Reply:
I think my workspace prettier setup might be clashing with the docs repo one...

Reply:
Not sure if that helps but I see ```js and this is not valid js, maybe switch to another code tag
```
PRISMA_FMT_BINARY=custom/my-custom-format-engine-unix
```

Reply:
Oh thats a relic, i didnt write that. But i didnt notice it either so will change. Good catch!